alibi_detect/cd/tensorflow/spot_the_diff.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+import tensorflow as tf
+from typing import Callable, Dict, Optional, Union
+from alibi_detect.cd.tensorflow.classifier import ClassifierDriftTF
+from alibi_detect.utils.tensorflow.data import TFDataset
+from alibi_detect.utils.tensorflow import GaussianRBF
+
+
+class SpotTheDiffDriftTF:
+    def __init__(
+            self,
+            x_ref: np.ndarray,
+            p_val: float = .05,
+            preprocess_x_ref: bool = True,
+            update_x_ref: Optional[Dict[str, int]] = None,
+            preprocess_fn: Optional[Callable] = None,
+            kernel: Optional[tf.keras.Model] = None,
+            n_diffs: int = 1,
+            initial_diffs: Optional[np.ndarray] = None,
```

Comment:
It's probably worth opening an issue highlighting best practice or providing intuition about the impact of both `n_diffs` and `initial_diffs` on the feature attributions obtained from the detector. Either in the docs or made clear via an example.

Reply:
Yeah I'll touch on it in the example notebook but would require further investigation to nail down.